### ISSUE TITLE
why it didn't use softmax in computing multichoice loss

### ISSUE THREAD

**Issue by eveliao** [huggingface/pytorch-openai-transformer-lm#51]
the input are just logits, not normalized by softmax, why can we directly compute the cross entropy loss with it and y?

https://github.com/huggingface/pytorch-openai-transformer-lm/blob/bfd8e0989c684b79b800a49f8d9b74e559298ec2/loss.py#L22

**Comment by eveliao** [comment]
cross entropy includes log_softmax function, sorry...
